pkgs/top-level/all-packages.nix

```diff
@@ -34754,6 +34756,10 @@ with pkgs;
 
   nix-simple-deploy = callPackage ../tools/package-management/nix-simple-deploy { };
 
+  nix-build-profiler = callPackage ../development/python-modules/nix-build-profiler {
+    inherit (python3Packages) buildPythonPackage psutil prefixed gnumake-tokenpool;
+  };
```

Comment:
Trying to run `nixpkgs-review` I get:

<pre><font color="#26A269">$ nix-env --option system x86_64-linux -f /home/kevincox/.cache/nixpkgs-review/pr-178171-1/nixpkgs -qaP --xml --out-path --show-trace --meta</font>
<font color="#C01C28"><b>error:</b></font> attribute &apos;<font color="#A347BA"><b>gnumake-tokenpool</b></font>&apos; missing

       <font color="#12488B"><b>at </b></font><font color="#A347BA"><b>/home/kevincox/.cache/nixpkgs-review/pr-178171-1/nixpkgs/pkgs/top-level/all-packages.nix:34759:86</b></font>:

        34758|
        34759|   nix-build-profiler = callPackage ../development/python-modules/nix-build-profiler {
             |                                                                                      <font color="#C01C28"><b>^</b></font>
        34760|     inherit (python3Packages) buildPythonPackage psutil prefixed gnumake-tokenpool;
</pre>

Reply:
thx. fixed by removing nix-build-profiler